assets/src/blocks/Carouselheader/CarouselHeader.js

```diff
@@ -80,26 +80,16 @@ export class CarouselHeader extends Component {
     return (
 
       <div>
-        <h3>{__('What style of carousel do you need?', 'p4ge')}</h3>
+        <h3>{__('Configure your carousel', 'p4ge')}</h3>
 
         <div>
-          <LayoutSelector
-            selectedOption={this.props.block_style}
-            onSelectedLayoutChange={this.props.onBlockStyleChange}
-            options={[
-              {
-                label: __('Zoom and slide to gray', 'p4ge'),
-                image: window.p4ge_vars.home + 'images/carousel-with-preview.png',
-                value: 'zoom-and-slide-to-gray',
-                help: __('This carousel provides a fancy transition, and a preview for the next slide in an oblique shape.')
-              }, {
-                label: __('Full width classic', 'p4ge'),
-                image: window.p4ge_vars.home + 'images/carousel-classic.png',
-                value: 'full-width-classic',
-                help: __('This is a full width slider with a classic look: big slides, fade transition, and no subheaders.'),
-              },
-            ]}
-          />
+          <div className='LayoutSelector'>
+          <label>
+            <div>{__('Full width classic', 'p4ge')}</div>
```

Comment:
Not sure we still need to put a name (and a [help text](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/324/files#diff-b1e24ef490bf7dce9f07bdeb1b988867R90)) if that's the only style available? 🤔